package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_cmake_doxygen</name>
+  <version>0.6.2</version>
```

Comment:
Since this is a new package in a new repository, I'd suggest version `0.1.0`.

Reply:
+1